lib/ui/view/edit_profile/edit_profile_viewmodel.dart

```diff
@@ -1,11 +1,222 @@
+import 'dart:convert';
+
+import 'package:dio/dio.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
 import 'package:hng/app/app.locator.dart';
+import 'package:hng/app/app.router.dart';
+import 'package:hng/general_widgets/app_toast.dart';
+
+import 'package:hng/package/base/server-request/api/http_api.dart';
+import 'package:hng/services/connectivity_service.dart';
+import 'package:hng/services/local_storage_services.dart';
+import 'package:hng/utilities/enums.dart';
+import 'package:hng/utilities/storage_keys.dart';
 import 'package:stacked/stacked.dart';
 import 'package:stacked_services/stacked_services.dart';
 
+ProfileModel stuff = new ProfileModel();
```

Comment:
Why is it outside the class? also why name a variable `stuff`, find a readable code name